ts/Series/Treegraph/TreegraphSeriesDefaults.ts

```diff
@@ -22,20 +22,19 @@ import type TreegraphSeriesOptions from './TreegraphSeriesOptions';
 import { Palette } from '../../Core/Color/Palettes';
 
 /* *
-*
-*  Constants
-*
-* */
+ *
+ *  Constants
+ *
+ * */
 
 /**
  * A treegraph series is a diagram, which shows a relation between ancestors
  * and descendants with a clear parent - child relation.
  * The best examples of the dataStructures, which best reflect this chart
  * are e.g. genealogy tree or directory structure.
  *
- * TODO change back the demo path
- * @sample highcharts/demo/treegraph-chart
- *          Treegraph Chart
```

Comment:
@bre1470  the idea was to change the sample's path back when PR #17600 gets merged. (apart from the fact, that there is a typo `highchart`, was this intended?)

Reply:
I think it is best if you fix the issue with treegraph samples. The warnings about invalid `@sample` will show when `npm run jsdoc` on master.